packages/shorebird_cli/lib/src/executables/powershell.dart

```diff
@@ -34,9 +35,14 @@ class Powershell {
 
   /// Returns the version string of the given executable file.
   Future<String> getExeVersionString(File exeFile) async {
+    // Reads the Windows ProductVersion for the provided executable using
+    // PowerShell's Get-Item cmdlet.
     final exePath = exeFile.path;
     final pwshCommand =
         "(Get-Item -Path '$exePath').VersionInfo.ProductVersion";
+    logger
+      ..detail('[powershell] Inspecting EXE for ProductVersion: $exePath')
+      ..detail('[powershell] Command: powershell.exe -Command $pwshCommand');
```

Comment:
I would move this `powershell.exe -Command $pswhCommand` log to the `pwsh` method if we want to keep it

Reply:
Got it. Moving it.